torch/_inductor/codegen/rocm/ck_universal_gemm_template.py

```diff
@@ -251,10 +280,20 @@ def render(self, kernel: ROCmTemplateKernel, op: "CKGemmOperation", **kwargs) ->
         template_buffer_node = kwargs.get("template_buffer_node", None)
         if template_buffer_node is not None:
             self.output_node = template_buffer_node
-        instance_definition, instance_type = self.emit_ck_instance(op)
         X, W = self.input_nodes[0], self.input_nodes[1]
         Y = self.output_node
-        Bias = None  # TBD support gemm_bias
+        Bias = self.input_nodes[2] if 3 == len(self.input_nodes) else None
+
+        op = copy.deepcopy(op)
+
+        if Bias:
```

Comment:
Maybe check `if Bias is not None` to be on the safe side? Also everywhere below.